rest/api_test.go

```diff
@@ -217,29 +217,33 @@ func TestCORSOrigin(t *testing.T) {
 	reqHeaders := map[string]string{
 		"Origin": "http://example.com",
 	}
-	response := rt.SendRequestWithHeaders("GET", "/{{.keyspace}}/", "", reqHeaders)
+	response := rt.SendRequestWithHeaders("GET", "/{{.db}}/", "", reqHeaders)
+	RequireStatus(t, response, http.StatusUnauthorized)
```

Comment:
Were these tests previously returning these statuses, or does that change in this PR?

Reply:
If you ask `{{.keyspace}}` it will return 400, which is how keyspace is resolved than db differently. I could leave this and test that it returns 400. 

This change is only a test clarification, I wanted to assert I wasn't changing the error code, but I can leave this out of this initial PR.

I added this code to make sure nothing was changed.